pkg/syscallfilter/syscallfilter_linux.go

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:build linux && amd64
-// +build linux,amd64
+//go:build (linux && arm64) || (linux && amd64) || (linux && riscv64)
+// +build linux,arm64 linux,amd64 linux,riscv64
```

Comment:
i don't think the strace package has riscv64 list of syscalls yet

Reply:
I tested only on amd64 and aarch64 myself, but we have this commit that adds syscall_linux_riscv64.go

```
tree e1a71a602b789bbf39cb380832b58cd5595a149a
parent 7491dfb2a6b7355035126708a43af943adf3ea21
author Ronald G Minnich <rminnich@gmail.com> Fri Jul 21 14:24:45 2023 -0700
committer ron minnich <rminnich@gmail.com> Thu Jul 27 14:20:33 2023 -0700

strace: add riscv64 to pkg and cmd

The system calls need to be checked. But I have been using this,
extensively, on riscv64.
```